frontend/src/components/regions/CardDetailsModal/ReviewsTable.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import ReviewCard from "./ReviewsSbonelo";
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: false
+  },
+});
+
+export default function ReviewsTable() {
```

Comment:
Rule number 1 of naming things: Call it what it is.

Is this a table?

Reply:
This has been fixed as well